projects/dsp-ui/src/lib/action/components/select-project/select-project.component.ts

```diff
@@ -0,0 +1,121 @@
+import { Component, EventEmitter, Inject, Input, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup } from '@angular/forms';
+import { ApiResponseData, ApiResponseError, KnoraApiConnection, ProjectsResponse, ReadProject } from '@dasch-swiss/dsp-js';
+import { Observable } from 'rxjs';
+import { map, startWith } from 'rxjs/operators';
+import { DspApiConnectionToken } from '../../../core';
+import { SortingService } from '../../services/sorting.service';
+
+@Component({
+    selector: 'dsp-select-project',
+    templateUrl: './select-project.component.html',
+    styleUrls: ['./select-project.component.scss']
+})
+export class SelectProjectComponent implements OnInit {
+
+    // list of projects
+    projects: ReadProject[];
+
+    // selected project
+    selectedProject: ReadProject;
```

Comment:
I have updated the code to return ReadProject object. But currently I am looping the complete projects array to get the selected project info. It would be good to have dictionary and not array for all projects to improve performance.